lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -18,13 +27,67 @@
                         courseID: this.courseID,
                         discussionTopicID: this.discussionTopicID,
                         canCreateComment: canPostToTeam,
-                        canCreateSubComment: canPostToTeam
+                        canCreateSubComment: canPostToTeam,
+                        country: this.model.get('country'),
+                        language: this.model.get('language'),
+                        membershipText: interpolate(
+                            // Translators: The following message displays the number of members on a team.
+                            ngettext(
+                                '%(member_count)s / %(max_member_count)s Member',
+                                '%(member_count)s / %(max_member_count)s Members',
+                                this.maxTeamSize
+                            ),
+                            {member_count: this.memberships.length, max_member_count: this.maxTeamSize}, true
+                        ),
+                        isMember: this.isUserMemberOfTeam(),
+                        hasCapacity: this.memberships.length < this.maxTeamSize,
+                        inviteLink: window.location +'?invite=true'
+
                     }));
                     this.discussionView = new TeamDiscussionView({
                         el: this.$('.discussion-module')
                     });
                     this.discussionView.render();
+
+                    this.renderTeamMembers();
                     return this;
+                },
+
+                renderTeamMembers: function() {
+                    var view = this;
+                    _.each(this.memberships, function(data) {
+                        $.ajax({
```

Comment:
Making separate requests just to get the profile data of members is clearly not the right way to go about it. I looked at the `TeamsDetailView` and it seems to be inlining the accounts data of the members. I have a feeling this should be the profile data of members. Can you ask the Cambridge team about this?